tools/lint/cmd.c

```diff
@@ -70,7 +70,7 @@ COMMAND commands[] = {
     },
     {
         "data", cmd_data_opt, cmd_data_dep, cmd_data_store, cmd_data_process, cmd_data_help, NULL,
-        "Load, validate and optionally print instance data", "d:ef:F:hmo:O:R:r:nt:x:"
+        "Load, validate and optionally print instance data", "d:ef:F:hmo:O:R:r:nt:x:ik:"
```

Comment:
`i` parameter seems redundant and a data type `ext` should be used just like for non-interactive params.